subgraphs/dopex/src/entities/snapshots.ts

```diff
@@ -178,11 +176,8 @@ export function takeLiquidityPoolDailySnapshot(
     prevContractsExpiredCount = prevPoolMetrics.contractsExpiredCount;
     prevContractsExercisedCount = prevPoolMetrics.contractsExercisedCount;
     prevContractsClosedCount = prevPoolMetrics.contractsClosedCount;
-    prevCumulativeOpenPositionCount =
-      prevPoolMetrics._cumulativeOpenPositionCount;
-    prevClosedPositionCount = prevPoolMetrics.closedPositionCount;
   } else if (pool._lastSnapshotDayID > INT_ZERO) {
-    log.error(
+    log.critical(
```

Comment:
I am thinking this should maybe only be an error. My reasoning is that there is something wrong and we should be notified, but the data isn't necessarily wrong so I don't think it should warrant a crash of the subgraph.

Open to thoughts on why it should be critical

Reply:
It was fixed before. The latest version is at https://github.com/messari/subgraphs/blob/aa3ec1a87ade6fbaf5381e591d863e48835f14ec/subgraphs/dopex/src/entities/snapshots.ts#L642

Reply:
Agree. Fixed at 60914eb148dff684e51c940de250b7c6ae30e1f0